src/screens/Classes.js

```diff
@@ -7,6 +7,7 @@ import theme from '../styles/theme.style.js';
 import ScreenContainer from '../containers/ScreenContainer';
 import { Title, Subtitle, TextBody } from '../containers/TextContainer.js';
 import MainContainer from '../containers/MainContainer.js';
+import TeamButton from '../components/AddButton.js';
```

Comment:
Might be better to call it something more specific, such as AddTeamButton, or something similar so that there isn't any other ambiguity with other buttons we might find on the teams page

Reply:
updated!

Reply:
thank you!